release.md

```diff
@@ -0,0 +1,167 @@
+# Ignition Release Process
+
+This documentation covers the releasing process done by the members of the
+Ignition/Gazebo team. It offers a general overview and documentation of the
+procedures and tools used and provide an step by step guide to do released.
+
+#### Table of Contents
+
+[General Overview](#general-overview)
+  * [Platforms Supported](#releasing-platforms-supported)
+  * [Infrastructure](#infrastructure)
+  * [Type of Releases](#type-of-releases)
+  * [Metadata for releasing](#metadata-for-releasing)
+  * [Versioning](#versioning)
+
+[Triggering a Release](#)
+
+[FAQ](#faq)
+
+## General overview
+
+Ignition development supports different ways of installing the software in the
+user system. This document covers the process of creating binaries for different
+distributions starting from a local copy of source code together with
+description and explanations of the context of preparing, creating and releasing
+new software versions.
+
+![general releasing context](releasing/images/general_context.png)
+
+### Releasing platforms supported
+
+The Ignition software supports a large variety of platforms where it can be
+built from source. A small subgroup of them is selected to be the officially
+supported for binaries distribution:
+
+ * **Linux:** .deb packages are being created for different Ubuntu and Debian
+   releases. Ubuntu LTS and Debian stable are officially supported.
+   Architectures supported are typically `amd64`, `arm64` and `armhf` but that
+   combination may not be available for all Ubuntu/Debian/Ignition releases.
+
+ * **MacOSX:** binaries for all active releases of MacOSX are usually created by
+   the Ignition development team under the support of
+   [Homebrew package manager](https://brew.sh/). Binary packages for Brew are
+   called `bottles`.
+
+ * **Windows:** currently *there is no binary distribution for Windows* that cover
+   all Ignition libraries although there is an ongoing effort for getting
+   binary packages inside
+   [conda-forge](https://github.com/conda-forge/staged-recipes/issues/13551)
+
+### Infrastructure
+
+The Ignition team uses different pieces of infrastructure to execute the
+releasing process:
+
+ * **build.osrfoundation.org**: main Jenkins server to orchestrate the whole
+   releasing process. It receives requests from Ignition developers to start
+   release process, schedule nightly builds, build the binary packages,
+   upload releasing artifacts to repositories, etc.
+
+ * **packages.osrfoundation.org**: Debian/Ubuntu .deb repositories.
+
+ * **osrf-distributions S3**: [cloud storage](https://osrf-distributions.s3.amazonaws.com/bottles-simulation)
```

Comment:
This link doesn't seem to work

Reply:
One thing that could be better clarified, either here or on a future PR, is the relationship between `packages.osrfoundation.org` and `osrf-distributions` (which isn't very clear to me).

Reply:
> One thing that could be better clarified, either here or on a future PR, is the relationship between `packages.osrfoundation.org` and `osrf-distributions` (which isn't very clear to me).

Let's try: a7d7ae7

> This link doesn't seem to work

Not sure if the S3 directories can be directly accessed, our own implementation in gazebosim tells me that they probably are not. a7d7ae7